overlord/booted.go

```diff
@@ -17,12 +17,14 @@
  *
  */
 
-package snappy
+package overlord
```

Comment:
Thanks, that is a great suggestion. I moved them into overlord/boot now. I renamed `UpdateRevisions` but the FirstBoot->InitialSetup seems to be slightly bigger so I would suggest a separate branch for this(?).